platforms/core-runtime/build-configuration/src/main/java/org/gradle/internal/buildconfiguration/tasks/DaemonJvmPropertiesAccessor.java

```diff
@@ -60,6 +60,10 @@ public JvmVendorSpec getVendor() {
         }
     }
 
+    public Boolean getNativeImageCapable() {
```

Comment:
:x: Why is this a `Boolean` instead of a `boolean`? It will never be `null`.

Reply:
Correct, fixing